solana/program_v2/programs/gateway_v2/src/instructions/admin/update_network.rs

```diff
@@ -24,6 +26,24 @@ pub struct UpdateNetworkData {
     pub fees: UpdateFees,
     /// The [`GatekeeperNetwork::auth_keys`].
     pub auth_keys: UpdateKeys,
+    /// The [`GatekeeperNetwork::network_features`].
+    pub network_features: Option<u32>,
```

Comment:
I don't see any reason to have this as an Option? e.g. it is an bitflag value that is driven by convention. meaning if we don't assign any value to a bit, we just don't care about it.

Not sure what the additional Option would bring us.